getting-started/templates/systemlink-values.yaml

```diff
@@ -131,6 +131,7 @@ global:
 ##
 ## Hostname of an external S3 provider.
 # <ATTENTION> To connect to an external S3 provider, set s3Host, s3Scheme, s3Port and s3Region.
+#             To use Azure Storage, ignore these values and reference Azure/azure_supplemental_values.yml.
```

Comment:
Any issues with having these defaults configured when deploying to Azure.

You could always split out an s3_supplemental_values configuration if needed.